client/jetpack-cloud/sections/partner-portal/unassign-license-dialog/style.scss

```diff
@@ -0,0 +1,56 @@
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
+
+.unassign-license-dialog {
+	.dialog__content {
+		max-width: 628px;
+
+		&, & p {
+			font-size: 1rem;
+		}
+	}
+
+	.dialog__action-buttons {
+		display: flex;
+		justify-content: flex-end;
+		align-items: center;
+		background: var( --studio-gray-0 );
+		border: 0;
+
+		a {
+			margin-right: auto;
+			font-weight: 600;
+			text-decoration: underline;
+		}
+	}
```

Comment:
These styles are a little outdated. Our current production version has a different style for the dialog components on small screens.

Such as this revoke dialog:
![image](https://user-images.githubusercontent.com/5550190/182512085-b92c959e-ea82-41b3-b792-97e574242d48.png)

I would suggest rebasing this branch and applying the updated styles from the revoke dialog to this dialog as well.

Making those styles to be shared somehow would be nice, maybe having a higher Dialog component to apply these shared styles.

Reply:
We could skip the creation of a higher Dialog component for now and have it as a maintenance task to be looked at in the future, as we currently just have these 2 dialogs.